packaging/homebrew/cassandra-cpp-driver.rb

```diff
@@ -3,13 +3,15 @@
 class CassandraCppDriver < Formula
   homepage "http://datastax.github.io/cpp-driver/"
   url "https://github.com/datastax/cpp-driver/archive/2.2.0-beta1.tar.gz"
-  sha1 "6159f1a5e31a044fb70567f849286572afa57174"
+  sha256 "5a7325057231b65ad36359fe07d37b0dfbf2a522fa6961138c5efbaa7b265831"
   version "2.2.0-beta1"
 
   head "git://github.com:datastax/cpp-driver.git", :branch => "master"
 
   depends_on "cmake" => :build
   depends_on "libuv"
+  depends_on "openssl"
+  depends_on "boost160"
```

Comment:
Why was boost added as a dependency? Only tests require boost and the formula doesn't build the tests.